code/modules/mob/living/carbon/human/species/vox.dm

```diff
@@ -173,9 +173,9 @@
 	eyes = "blank_eyes"
 
 	species_traits = list(NO_SCAN, NO_BLOOD, NO_PAIN, IS_WHITELISTED)
-	dies_at_threshold = TRUE
+	clothing_flags = 0 //IDK if you've ever seen underwear on an Armalis, but it ain't pretty.
```

Comment:
Lol.

Reply:
Haha yeah I couldn't help but check and it was mega wack so I figured I'd fix it. I should've probably worded it like "seen an Armalis in their underwear" but oh well, hindsight 😂👍